lab-taylor-melanie-ken/__test__/transform.test.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
+
+const fs = require('fs');
+const bitMapWriter = require('../lib/bitmap-writer.js');
+const constructor = require('../lib/bitmap-constructor.js');
+const transform = require('../lib/transform.js');
+require('jest');
+
+let bm = fs.readFileSync(`${__dirname}/../data/palette-bitmap.bmp`);
+// console.log('data in test:', bm);
```

Comment:
A commented out console log is better than an extraneous one, but a deleted console.log is even better!